e2e/cypress/integration/messaging/typing_should_show_up_when_editing_spec.js

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```

Comment:
Please remove since we'll put it first on staging for several run and then move to production test one we found it to be stable.
```suggestion
```

Reply:
Thanks for your comments @saturninoabril 
I removed this line, but I'm not sure what to change in package-lock.json. If you could please clarify that, I'll change it asap :)